src/axom/core/tests/core_array.hpp

```diff
@@ -1931,4 +1931,18 @@ TEST(core_array, checkVariadicCtors)
   Array<int, 3> arr12(s, s, s);
 }
 
+//------------------------------------------------------------------------------
+
+TEST(core_array, check_subspan_range)
+{
+  int m = 10;
+  int n = 3;
+  Array<int> arr(m);
+  ArrayView<int> arrv1(arr);
+  ArrayView<int> arrv2 = arrv1.subspan(n);
```

Comment:
Should we also add a test for `subspan()` with two parameters? Or is that tested elsewhere?

Reply:
It's not tested elsewhere.  I'll add.